footprint/avr/run_suite.py

```
@@ -0,0 +1,209 @@
+TARGET_DIR = os.path.join(tempfile.gettempdir(), "rsa_footprint_avr")
+SYSROOT = subprocess.run(
+    ["rustc", "--print", "sysroot"], capture_output=True, text=True, check=True
+).stdout.strip()
+
+
+def run_cmd(args, timeout=TIMEOUT_RUN, **kwargs):
+    """Run a command, return (returncode, stdout, stderr)."""
+    env = os.environ.copy()
+    env.setdefault("CARGO_TARGET_DIR", TARGET_DIR)
+    env["PATH"] = r"E:\m\depot_tools;" + env.get("PATH", "")
+    result = subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)